api/Window.json

```diff
@@ -8468,19 +8450,19 @@
             ],
             "firefox": [
               {
-                "version_added": true
+                "version_added": "1"
```

Comment:
Is the "alternative_name" `pageXOffset` what you tested here? http://mdn-bcd-collector.appspot.com/tests/api/Window/pageXOffset exists, but our update scripts don't handle alternative names and prefixes correctly yet...

Reply:
I tested both `scrollX` and `pageXOffset` to make sure both of them were present!

Reply:
Thank you!